pow/src/test/java/tech/pegasys/artemis/pow/MinimumGenesisTimeBlockFinderTest.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.pow;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.primitives.UnsignedLong;
+import java.math.BigInteger;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import tech.pegasys.artemis.util.async.SafeFuture;
+import tech.pegasys.artemis.util.config.Constants;
+
+public class MinimumGenesisTimeBlockFinderTest {
+
+  private Eth1Provider eth1Provider;
+
+  private MinimumGenesisTimeBlockFinder minimumGenesisTimeBlockFinder;
+  //  private PublishSubject<EthBlock.Block> blockPublisher;
+
+  @BeforeEach
+  void setUp() {
+    eth1Provider = mock(Eth1Provider.class);
+
+    minimumGenesisTimeBlockFinder = new MinimumGenesisTimeBlockFinder(eth1Provider);
+
+    //    blockPublisher = mockFlowablePublisher();
+
+    Constants.MIN_GENESIS_DELAY = 1;
```

Comment:
Can you add a `@TearDown` to reset `Constants` to the minimal config again so other tests don't get caught out please.

Reply:
Done.